src/Go/ui/GoHistoryPage.tsx

```diff
@@ -21,7 +21,7 @@ export const GoHistoryPage = (): React.ReactElement => {
   const priorBoard = Player.go.previousGameFinalBoardState ?? getNewBoardState(7);
   const score = getScore(priorBoard);
   const opponent = priorBoard.ai;
-  const opponentsToShow = showWorldDemon() ? [...opponentList, opponents.w0r1d_d43m0n] : opponentList;
+  const opponentsToShow = getOpponentList(showWorldDemon());
```

Comment:
I don't think you ever use `getOpponentList` with a first arg that *isn't* `showWorldDaemon`. You could move the helper to goAI (constants is a strange place for a function anyway) and eliminate the first arg.

Reply:
Good call, updated